modules/statics/test/unit/coins.ts

```diff
@@ -148,6 +148,14 @@ const custodyFeatures: Record<string, { features: CoinFeature[] }> = {
       CoinFeature.CUSTODY_BITGO_FRANKFURT,
     ],
   },
+  hteth: {
```

Comment:
nit: can we add hteth above only with gteth. Or the idea is to remove gteth later.